README.md

```diff
@@ -1,111 +1,148 @@
-# IBM CIS Terraform Module
+# IBM Cloud Internet Services (CIS) Module
 
-Collection of modules that make it easier to provision Cloud Internet Services and its resources on IBM Cloud Platform:
+This module provisions an IBM Cloud Internet Services (CIS) instance.
 
-## Compatibility
+It includes the sub-modules that allows to add the following into the CIS instance.
```

Comment:
```suggestion
The module includes submodules to add the following features to the CIS instance.
```

I think you can move this 2nd sentence to line 3 to be part of the same paragraph.

Reply:
done